docs/config.md

```diff
@@ -129,12 +129,12 @@ change_id = "#ff1525"
 ```
 
 If you use a string value for a color, as in the examples above, it will be used
-for the foreground color. You can also set the background color, or make the
-text bold, italic, or underlined. For that, you need to use a table:
+for the foreground color. You can also set the background color, reverse colors,
```

Comment:
nit: maybe better to say "swap foreground and background colors"? (it's unclear what will be reversed.)

Reply:
Good idea, I've adjusted the description (while keeping the word "reverse" to associate it with the keyword that's used).